runner/package.json

```diff
@@ -28,6 +28,8 @@
   },
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "7.10.1",
+    "@babel/plugin-proposal-private-methods": "^7.10.1",
```

Comment:
these should be dev deps. it'll stop flow-typed from trying to stub it too

Reply:
Fixed